models/magiceden/solana/magiceden_solana_trades.sql

```diff
@@ -0,0 +1,231 @@
+{{
+    config(
+        schema = 'magiceden_solana'
+        , tags = ['dunesql']
+        , alias = alias('trades')
+        ,materialized = 'incremental'
+        ,file_format = 'delta'
+        ,incremental_strategy = 'merge'
+        ,unique_key = ['unique_trade_id','block_slot']
```

Comment:
We've been pushing to phase out the use of `unique_trade_id` for the merge key. 
Can we make this a minimal set of unique columns?
For reference on evm models this is `['block_number','tx_hash','sub_tx_trade_id']` where the `sub_tx_trade_id` is a unique number for when multiple trades happen in 1 tx, often we use the `evt_index` here or use a `row_number()` partitioned on `tx_hash` and ordered by `trace_address`.

From looking at this model I think the equivalent would be something like `['block_slot', 'tx_id', 'call/log_order']`?
If possible. let's adapt the same `sub_tx_trade_id` column name.

Reply:
In the overall `trades_solana` model this unique_key would then become `['project', 'block_slot', 'tx_id', 'sub_tx_trade_id']`

Reply:
after that we should add this test to the model schema files to ensure we have a unique set:
```
   tests:
     - dbt_utils.unique_combination_of_columns:
          combination_of_columns:
            - block_slot
            - tx_id
            - sub_tx_trade_id
```